chaoscenter/authentication/api/handlers/rest/user_handlers.go

```diff
@@ -123,13 +140,26 @@ func UpdateUser(service services.ApplicationService) gin.HandlerFunc {
 
 		// Checking if password is updated
 		if userRequest.Password != "" {
+			err := utils.ValidateStrictPassword(userRequest.Password)
+			if err != nil {
+				c.JSON(utils.ErrorStatusCodes[utils.ErrStrictPasswordPolicyViolation], presenter.CreateErrorResponse(utils.ErrStrictPasswordPolicyViolation))
+				return
+			}
 			hashedPassword, err := bcrypt.GenerateFromPassword([]byte(userRequest.Password), utils.PasswordEncryptionCost)
 			if err != nil {
 				return
 			}
 			userRequest.Password = string(hashedPassword)
 		}
 
+		if userRequest.Name != "" {
+			err = utils.ValidateStrictUsername(userRequest.Name)
```

Comment:
Why are we adding username validation for name field?

Reply:
oh, my bad I thought we want validation for the name. will be removing it.